st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -83,7 +83,10 @@ class ActionExecutionsControllerMixin(BaseRestControllerMixin):
     mandatory_include_fields_retrieve = [
         'action.parameters',
         'runner.runner_parameters',
-        'parameters'
+        'parameters',
+        # necessary for ActionExecutionDB to determine permissions in enterprise ldap
```

Comment:
In theory, we could only include that if rbac is enabled, but since those attributes are quite small, it doesn't hurt to include them in all the responses...